### DISCUSSION TITLE
Unable to apply with educational E-Mail

### DISCUSSION THREAD

**Discussion by noah-s-k** [community/community#55619] [GitHub Education]
Greetings,

I do have an educational E-Mail from my school network, but GitHub doesn't seem to identify it as one. I found one possible explanation myself: 
The schools homepage lies under the [SCHOOLNAME].de domain. This domain is also listed in the drop-down menu on the GitHub education page as the schools domain. BUT all E-Mails in the school network run under [SCHOOLNAME].net not .de 

I hope that changing the domain for the school can solve this problem. I could also provide prove to under which the E-Mails run, as there's a Page on the .de domain explaining it.

Have a nice day,
Noah

**Comment by Calosis** [comment]
Exactly the same issue. It's pathetic that there has been no support from any Github staff member as well. I had opened a ticket, waited a month, and then my ticket was just closed. Told to post about the issue here, another month, and still NOTHING.

**Comment by samus-aran** [comment]
Hi @mrflolo 👋 

Re-reading both discussions I appreciate all your patience and time responding. We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. In this particular case, yes this needs ourselves to step in and review the school details. I've raised this with the internal team and we'll update this thread with any updates!

**Reply by noah-s-k** [reply]
Hi @samus-aran,

Glad to hear something back even if it's after some time ^^. If needed I can share the explicit page on the school homepage where the E-Mail domains are declared. I didn't send the link in the original post as per the Community Guidelines no school names or other personal information should be listed. But I've written all important things in the support ticket #2205125 I opened some time ago.

Have a nice Day,
Noah

**Reply by samus-aran** [reply]
That's understandable. I'm able to find your raised support ticket so I don't need any additional information 👍

**Reply by parker02311** [reply]
Hi @samus-aran 

I am also having the same issue and have also already had to open a support ticket about this. My original ticket had it fixed but for some reason now when I go back to do it again it appears that the fix was reverted. On top of this because of the slow support response I’ve already lost benefits without being able to reactivate them. My currently open ticket is is `2256946`. 

Mahalo
